development/tools/cmd/githubrelease/main.go

```diff
@@ -16,8 +16,8 @@ import (
 
 var (
 	targetCommit           = flag.String("targetCommit", "", "Target commitish [Required]")
-	bucketName             = flag.String("bucketName", "kyma-prow-artifacts", "Google bucket name where artifacts are stored [Optional]")
 	kymaComponents         = flag.String("kymaComponents", "kyma-components.yaml", "Filename for kyma-components.yaml file. [Optional]")
+	kymaComponentsPath     = flag.String("kymaComponentsPath", "installation/resources/components.yaml", "File path for components.yaml file. [Optional]")
```

Comment:
Those 2 flags should have the same purpose. If its possible consolidate them into one. Since you removed bucket usage conpletely, i don't see why would you want to keep 2 flags here

Reply:
fixed